### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by cpiemontese**

### CODE REVIEW

**Review by oliverbrowneprima — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,40 +5,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Next]
 
+## [0.6.0] - 2023-05-10
```
Comment (cpiemontese) [inline CHANGELOG.md:8]:
```suggestion
## [0.6.0] - 2023-05-15
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,40 +5,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Next]
 
+## [0.6.0] - 2023-05-15
+
 ### Added
 
 - `dev` and `live` feature sets
+- ⚠️️ added mandatory `country` field
```
Comment (cottinisimone) [inline CHANGELOG.md:13]:
```suggestion
- ⚠️️ Added mandatory `country` field
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,40 +5,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Removed the default `prima-logger` feature. This is a non-breaking change since the library already failed to compile without it.
 
 ## [0.5.0] - 2022-07-04
 
 ### Changed
+
 - SubscriberConfig env field value changed from string to enumeration (Environment)
 - Update dependencies  
-- ⚠️  Increase the minimum rust version to 1.57.0
+- ⚠️ Increase the minimum rust version to 1.57.0
```
Comment (cottinisimone) [inline CHANGELOG.md:36]: Being that everytime we state this with the past can we change this to:
```suggestion
- ⚠️ Increased the minimum rust version to 1.57.0
```
?

**Review by cottinisimone — COMMENTED** [review]
Minor grammar-nazi stuff

**Review by cottinisimone — APPROVED** [review]

**Pull request merged**